installer/bundle/bundle_skel.sh

```diff
@@ -387,6 +411,55 @@ install_if_program_does_not_exist_on_system()
     fi
 }
 
+install_troubleshooter()
+{
+    echo "----- Installing troubleshooter -----"
+
+    # create temp directory
+    mkdir -p $TST_EXTRACT_DIR
+    cd $TST_EXTRACT_DIR
+
+    # grab tst bundle
+    wget $TST_PKG > /dev/null 2>&1
+    if [ $? -ne 0 ]; then
+        echo "Error downloading troubleshooter. To install it manually, please go to the below link:"
```

Comment:
Add in comment about how it will try to install with OMS install

Reply:
Fixed in these places:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/installer/bundle/bundle_skel.sh#L425
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/installer/bundle/bundle_skel.sh#L433
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/installer/bundle/bundle_skel.sh#L456